django/test/utils.py

```diff
@@ -359,6 +359,9 @@ def teardown_databases(old_config, verbosity, parallel=0, keepdb=False):
     """Destroy all the non-mirror databases."""
     for connection, old_name, destroy in old_config:
         if destroy:
+            # TODO: Hack :/
```

Comment:
Any reason why this can't be moved to the Postgres backend `destroy_test_db` method?

Reply:
Nothing specific no, should be doable :+1:

Reply:
Got rid of it completely, couldn't find the reason why it was there :)

Reply:
Oh I found it. When tests do fail it leaves the pool open. I wonder if we should add a hook to Django to prepare the base backend for something like `close_pool`

Reply:
Does this hook already exist in `destroy_test_db`? Why can't it be moved there?

Reply:
because there are more locations where I might wanna do some closing, like here: https://github.com/django/django/blob/d1be05b3e9209fd0787841c71a95819d81061187/django/db/backends/base/creation.py#L64

But yes, for now I will probably add it inside `destroy_test_db`.

Reply:
I made it postgres specific in `_destroy_test_db` for now.